examples/main.go

```diff
@@ -7,6 +7,7 @@ import (
 	"fmt"
 	"time"
 
+	"github.com/optimizely/go-sdk"
 	"github.com/optimizely/go-sdk/pkg/client"
 	"github.com/optimizely/go-sdk/pkg/entities"
```

Comment:
This is tricky because when imported, we import the namespace "optimizely", which is the package name and we would be referring to this import via `optimizely.xxx`

Reply:
I think @mikecdavis is talking about `"github.com/optimizely/go-sdk/pkg/entities"`, this is unused.